barretenberg/cpp/src/barretenberg/proof_system/relations/relation_parameters.hpp

```diff
@@ -20,6 +22,12 @@ template <typename FF> struct RelationParameters {
     // We can remove this by modifying the relation, but increases complexity
     FF eccvm_set_permutation_delta = 0;
     std::array<FF, NUM_BINARY_LIMBS_IN_GOBLIN_TRANSLATOR> accumulated_result = { FF(0) }; // Goblin Translator
+    std::array<FF, NUM_BINARY_LIMBS_IN_GOBLIN_TRANSLATOR + NUM_NATIVE_LIMBS_IN_GOBLIN_TRANSLATOR> evaluation_input_x = {
```

Comment:
I think these are not guaranteed to initialize to zero, rather, I think you might be only initializing the first element to zero. Cf "aggregate initialization" in C++ docs.

Reply:
Yes, probably (I'm used to this working in C, but it is better to be safe, since we don't  have an MSAN workflow yet)